at_secondary/at_secondary_server/lib/src/verb/handler/scan_verb_handler.dart

```diff
@@ -122,27 +133,33 @@ class ScanVerbHandler extends AbstractVerbHandler {
           key.toString().startsWith('privatekey:') ||
           key.toString().startsWith('public:_') ||
           key.toString().startsWith('private:'));
-      keyString = keys.toString();
+      keysList = keys;
     } else {
-      //When pol is performed, display keys that are private to the atsign.
+      // When pol is performed, display keys that are private to the atsign.
       if (atConnectionMetadata.isPolAuthenticated) {
         keys.removeWhere((test) =>
             (test
                     .toString()
                     .startsWith('${atConnectionMetadata.fromAtSign}:') ==
                 false) ||
             test.toString().startsWith('public:_'));
-        keyString = keys
-            .toString()
-            .replaceAll('${atConnectionMetadata.fromAtSign}:', '');
+        for (var key in keys) {
```

Comment:
Please explain how the above test identifies "keys that are private to the atsign". Saves time for the next reader of this code.

Consider 
1) Making this code a little more readable by extracting the test into its own function so the code could instead read something like
```
keys.removeWhere((key) => isPrivateKeyForAtSign(key, atConnectionMetadata.fromAtSign));
```

2) If exactly this same test is made elsewhere in the code base, consider eliminating the duplicate code by having those other places also use `isPrivateKeyForAtSign (String key, String atSign)`

Reply:
This change has been done @gkc . Please verify it.

Reply:
Thanks Minnu. Looks good once you've addressed Murali's latest comments.